pallets/types/src/traits.rs

```diff
@@ -322,3 +325,24 @@ impl<AssetId, Balance> BalancePrecisionConverter<AssetId, Balance, Balance> for
 pub trait TimepointProvider {
     fn get_timepoint() -> GenericTimepoint;
 }
+
+pub trait BridgeAssetLocker<AccountId> {
+    type AssetId: Parameter + MaybeSerializeDeserialize;
+    type Balance: Parameter + AtLeast32BitUnsigned + MaybeSerializeDeserialize;
+
+    fn lock_asset(
+        network_id: GenericNetworkId,
+        asset_kind: AssetKind,
+        who: AccountId,
+        asset_id: Self::AssetId,
+        amount: Self::Balance,
```

Comment:
```suggestion
        asset_kind: &AssetKind,
        who: &AccountId,
        asset_id: &Self::AssetId,
        amount: &Self::Balance,
```
Is it better to pass by reference and not to clone on every call? As I understand, the method doesn't require ownership of the values. But I don't know best practice, if you know. could you provide me a link?